docs/farming-&-staking/farming/advanced-cli/cli-tips.mdx

```diff
@@ -45,46 +45,31 @@ In certain situations, especially when the farmer terminates unexpectedly or enc
 
 Ensure to replace /path/to/your/plot with your actual plot path. Use this command cautiously as it modifies the plot state to recover from errors.
 
-### Using a Custom Path
-You can set a custom path for your node & farmer to use if you want to use an external hard drive, or set a custom path from the default. You can set the node and farmer to different directories if you would like.
-
-<Tabs>
-<TabItem value="node" label="1.🚩 Node">
-
-#### Set Node Custom Path.
-To set your node to use a custom path all you will need to do is add the `--base-path` parameter.
-
-```bash
-# start node and store data in `NODE_DATA_PATH` instead of default location
-./NODE_FILE_NAME --base-path NODE_DATA_PATH --chain gemini-3g ...`
-```
-</TabItem>
-</Tabs>
 
 ### Switching to a new snapshot from older/different versions of Subspace
 
 :::info
-Unless specifically mentioned by the Development team you should **NOT** have to wipe & purge your configuration on new releases.
+Unless specifically mentioned by the Development team you should **NOT** have to wipe your configuration on new releases.
 :::
 
 In general you should be able to download the latest release, and re-start the Node & Farmer with the same commands as you started to prior version with no errors.
 
-There are some cases where version updates will cause issue with your Node & Farmer and you may have to wipe & purge your node, typically when errors occur. If you have any issues you can always check our [Forums](https://forums.subspace.network) and hop in our [Discord](https://discord.gg/subspace-network) Server to ask for help.
+There are some cases where version updates will cause issue with your Node & Farmer and you may have to wipe your node, typically when errors occur. If you have any issues you can always check our [Forums](https://forums.subspace.network) and hop in our [Discord](https://discord.gg/subspace-network) Server to ask for help.
 
-### Wipe & Purge
+### Wipe
 
 If you were running a node previously, and want to switch to a new network, please perform these steps and then follow the guide again:
 ```bash
 # Replace `FARMER_FILE_NAME` with the name of the farmer file you downloaded from releases
 ./FARMER_FILE_NAME wipe PATH_TO_FARM
 # Replace `NODE_FILE_NAME` with the name of the node file you downloaded from releases
-./NODE_FILE_NAME purge-chain --chain gemini-3g
+./NODE_FILE_NAME wipe PATH_TO_NODE
```

Comment:
Is `PATH_TO_NODE` the value we use for `--base-path`?

Reply:
Yes, it is the same value, just positional argument in this case rather than named option

Reply:
It's named inconsistently in different places. And `PATH_TO_NODE_DATA` or even `NODE_DATA_PATH` feels like a better explanation.

Just checked and we use the latter elsewhere. I think we should standardise on that.